web-animations-1/Overview.bs

```diff
@@ -924,6 +929,32 @@ matching condition from below:
 
 </div>
 
+### The current phase of an animation ### {#the-current-phase-of-an-animation}
+
+<a>Animations</a> provide a <a>timeline phase</a> to their <a>associated
+effect</a> called the animation's <dfn>current phase</dfn>.
```

Comment:
Again, I think we should avoid referring to "timeline phase" here. Perhaps something like:

> In addition to their _current time_, _animations_ also provide their _associated effect_ with a **_current phase_**. This is used in determining the animation effect's _effect phase_ in certain boundary conditions.

We should also probably add:

> An animation's _current phase_ has the same range of values as the _timeline phase_.